app/build.gradle

```diff
@@ -97,7 +101,14 @@ dependencies {
     compile "com.android.support:recyclerview-v7:${rootProject.ext.androidSupportSdkVersion}"
     compile "com.squareup.leakcanary:leakcanary-android:1.3.1"
 
-    apt 'com.google.dagger:dagger-compiler:2.0.1'
+    apt 'com.google.dagger:dagger-compiler:2.5'
+    apt "com.ryanharter.auto.value:auto-value-gson:0.3.1"
+    apt "com.google.auto.value:auto-value:1.2"
+    // used for AndroidStudio sync
+    provided "com.google.auto.value:auto-value:1.2"
+    // to make OkBuck work with SqlDelight
+    provided 'com.github.okbuilds:SqlDelightBin:0.4.3'
```

Comment:
Hmm that sucks. Can we actually create a jar that can put the gradle plugin on the classpath and just call the public api to do what it needs to. That way we can switch out the sqldelight plugin jar with the version from gradle and still have the same result. Lesser maintanence for us too. Similar to how we do retrolambda